js/views/PageNav.js

```diff
@@ -67,6 +66,10 @@ export default class extends View {
     window.history.forward();
   }
 
+  navReload() {
+    remote.getCurrentWindow().reload();
```

Comment:
When I run this, I get:

```
PageNav.js [sm]:70 Uncaught TypeError: _electron2.default.getCurrentWindow is not a function
```

An alternate way to run this (which would save you an import) would be to just do `location.reload();`.